content/java/concepts/hashmap/hashmap.md

```diff
@@ -181,3 +181,37 @@ Jeanette
 Ben
 Lily
 ```
+
+## Traversing a `HashMap` using `map.entrySet()`
+
+Alternatively, Map.entry() can be used to return a `set` of key-value pairs. This might be more efficient in some cases where it's needed to access the key and the value.
```

Comment:
```suggestion
Alternatively, `Map.Entry()` can be used to return a `set` of key-value pairs. This might be more efficient in some cases where it's needed to access the key and the value.
```